Dockerfile

```diff
@@ -46,7 +46,7 @@ COPY tests/ tests/
 COPY pytest.ini pytest.ini
 COPY scripts/ scripts/
 COPY tutorials/ tutorials/
-COPY experiment_config experiment_config/
+COPY model_config model_config/
```

Comment:
What do you think of `training_config` here?  I think it's a little more clear.

Reply:
I don't care.  That works for me.